docs/content/preview/explore/linear-scalability/horizontal-vs-vertical-scaling.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Horizontal vs Vertical Scaling
+headerTitle: Horizontal vs vertical scaling
+linkTitle: Horizontal vs vertical
+description: Differences between horizontal and vertical scaling
+headcontent: Understand the differences between horizontal and vertical scaling
+menu:
+  preview:
+    identifier: scalability-horizontal-vs-vertical
+    parent: explore-scalability
+    weight: 5
+rightNav:
+  hideH3: true
+type: docs
+---
+
+After you have decided to scale your system, there are 2 common ways to scale, namely **vertical** and **horizontal**. YugabyteDB supports both. Because YugabyteDB is distributed, scaling is operationally straightforward and performed without any service disruption.
```

Comment:
```suggestion
After you have decided to scale your system, there are 2 common ways to scale, namely **vertical** and **horizontal**. YugabyteDB supports both. Because YugabyteDB is distributed, scaling is operationally straightforward and performed without any service disruption, regardless of which method you choose.
```